components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/dao/ConfigurationDAO.java

```diff
@@ -115,6 +115,17 @@ Resource getResourceByName(int tenantId, String resourceTypeId, String name)
      */
     void addResource(Resource resource) throws ConfigurationManagementException;
 
+    /**
```

Comment:
Formatting check all method comments

Reply:
Checked all places. There is no formatting issue.